src/openrct2/editor.c

```diff
@@ -521,7 +521,7 @@ bool editor_check_park()
 		}
 	}
 
-	if (gPeepSpawns[0].x == 0xFFFF && gPeepSpawns[1].x == 0xFFFF) {
+	if (gPeepSpawns[0].x == UINT16_MAX && gPeepSpawns[1].x == UINT16_MAX) {
```

Comment:
Yeah, I switched it to UINT16_MAX to match stuff like https://github.com/OpenRCT2/OpenRCT2/pull/5119/files#diff-38e5648eebb32758cb93a0071f9df207R367